src/UnityMvvmToolkit.UnityPackage/Assets/Plugins/UnityMvvmToolkit/Runtime/UITK/BindableUIElements/BindableListView.cs

```diff
@@ -90,7 +90,7 @@ private void BindItem(VisualElement item, int index)
 
         private void UnbindItem(VisualElement item, int index)
         {
-            UnbindItem(item, index, _itemsSource.Value[index], _objectProvider);
+            item.ResetBindingContext(_objectProvider, true);
```

Comment:
Not the best solution. I'll have a fix. Will be published later.

Reply:
@CunningFox146 Replace `BindItem` and `UnbindItem` with this:

```csharp
private void BindItem(VisualElement item, int index)
{
    if (index >= 0 && index < itemsSource.Count)
    {
        BindItem(item, index, _itemsSource.Value[index], _objectProvider);
    }
    else
    {
        BindItem(item, index, default, _objectProvider);
    }
}

private void UnbindItem(VisualElement item, int index)
{
    if (index >= 0 && index < itemsSource.Count)
    {
        UnbindItem(item, index, _itemsSource.Value[index], _objectProvider);
    }
    else
    {
        UnbindItem(item, index, default, _objectProvider);
    }
}
```

And I'll merge the fix.

Reply:
Don't forget to specify the issue like - "Fix #11".